### DISCUSSION TITLE
Personal Einsatzplanung

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3159] [Questions & Answers]
Hallo zusammen,

kennt jemand eine Möglichkeit in Kimai eine Personaleinsatzplanung durchzuführen und eine Auswertung auf Mitarbeiterebene zu machen mit der man die Verfügbarkeit der Mitarbeiter in den nächsten Monaten sehen könnte ? Es gibt Tasks und ich kann auch Urlaube eintragen. Eine Übersicht welcher Mitarbeiter für was wieviel in welchem Projekt verplant ist habe ich nicht finden können.

**Comment by kevinpapst** [comment]
Tasks liefert noch einen Report zur verplanten Auslastung mit:
https://demo-plugins.kimai.org/en/reporting/assigned-tasks

Ist aber weit entfernt von einer echten Personalplanung. Dennoch eine Möglichkeit einen groben Forecast abzugeben.

**Reply by jens-2305** [reply]
Danke für deine Antwort, ja das ist leider nicht ausreichend. Ich suche eine Möglichkeit das z.B. monatsweise machen zu können um zu sehen wie viele Stunden der jeweilige Mitarbeiter noch verfügbar wäre.
Kennst du jemanden der Kimai mit einer Personalplanung betreibt und z.B. die Mitarbeiter und Projekte mit Kimai synchronisiert ? Eigentlich habe ich in Kimai ja alles was ich brauche und würde gerne doppelte Pflege vermeiden.

**Reply by FAB-gGmbH** [reply]
> Danke für deine Antwort, ja das ist leider nicht ausreichend. Ich suche eine Möglichkeit das z.B. monatsweise machen zu können um zu sehen wie viele Stunden der jeweilige Mitarbeiter noch verfügbar wäre. Kennst du jemanden der Kimai mit einer Personalplanung betreibt und z.B. die Mitarbeiter und Projekte mit Kimai synchronisiert ? Eigentlich habe ich in Kimai ja alles was ich brauche und würde gerne doppelte Pflege vermeiden.

Hallo jens-2305,

wir haben tatsächlich das gleiche Anliegen, also eine Einsatzplanung in Kimai. Bist Du schon weitergekommen bzw. hast Du jemanden gefunden? Oder habt Ihr das für euch anders lösen können? Danke im Voraus und Grüße

**Reply by jens-2305** [reply]
Hi,
nein leider habe ich hier aktuell noch keine Lösung.
